orderbook/src/orderbook.rs

```diff
@@ -90,6 +91,17 @@ impl Orderbook {
         Ok(orders)
     }
 
+    pub async fn get_solvable_orders(&self) -> Result<Vec<Order>> {
+        let filter = OrderFilter {
+            min_valid_to: now_in_epoch_seconds(),
+            exclude_fully_executed: true,
+            exclude_invalidated: true,
+            exclude_insufficient_balance: true,
```

Comment:
Right, I see its all three flags equal true. I guess this makes a bit more sense to have a new route for it.